src/pages/Settings.vue

```diff
@@ -337,6 +360,52 @@ export default {
             this.$root.storage().removeItem("token");
         },
 
+        downloadBackup() {
+            let time = dayjs().format("YYYY_MM_DD-hh_mm_ss");
+            let fileName = `Uptime_Kuma_Backup_${time}.json`;
+            let monitorList = Object.values(this.$root.monitorList);
+            let exportData = {
+                version: this.$root.info.version,
+                notificationList: this.$root.notificationList,
+                monitorList: monitorList,
+            }
+            exportData = JSON.stringify(exportData);
+            let downloadItem = document.createElement("a");
```

Comment:
checked, exportData is ok, should be the problem of downloadItem, which remove all spaces unexpectedly.

Reply:
Thank you, should be fixed with [acf6f18](https://github.com/louislam/uptime-kuma/pull/306/commits/acf6f1883b829449cd0883185b301f3e599ecb39)